kafka-webview-ui/src/main/resources/schema/migration/h2/V1__init.sql

```diff
@@ -0,0 +1,109 @@
+
+create table if not exists `user` (
```

Comment:
Is this the flyway stuff? Does this get autogenerated or do you write this?

One note, for consistency sake you might want to match the casing on the other statements.